README.md

```diff
@@ -49,6 +49,18 @@ To generate a new revision after you've updated the models:
 2. Check the `/alembic/versions/` directory for the new revision and verify that it does what you want it to
 3. Run this from the root of the project: `DB_CONNECTION_STRING='postgresql://user:password@host:port/db_name' alembic upgrade head`
```

Comment:
🔧 Ah do we want to specify running `upgrade` once for each database?

Reply:
YUP good call